lib/x11regressiontest.pm

```diff
@@ -99,23 +99,14 @@ sub cleanup_libreoffice_specified_file {
 
 # cleanup libreoffice recent open file to make sure libreoffice clean
 sub cleanup_libreoffice_recent_file {
+    x11_start_program('xterm');
+    assert_script_run("rm -rf /home/$username/.config/libreoffice/");
```

Comment:
Isn't this removing actually much more than libreoffice does when clear list of recent documents? I just wonder if it can lead to some unexpected behavior because we remove some config files.

Reply:
Thanks for Comment.:) I had same concern about it, so I had done more test about this config directory before I modified this case. This directory was been created when libreoffice lanuched in the first time. And libreoffice will check this directory every time, the directory will be created again if it is not exist. So I think it would not lead an unexpected behaivor.

Reply:
Makes sense. Thanks for the explanation ;)